src/app/_components/DontHaveAnAccount/DontHaveAnAccount.tsx

```diff
@@ -0,0 +1,17 @@
+import { Box, Typography, alpha } from "@mui/material";
+
+const DontHaveAnAccount = (): JSX.Element => {
```

Comment:
There should be callback function available in the props that gets called when the `Sign up` link is clicked on. So please go ahead and define an `interface` for props and add a function prop in it.

Reply:
Resolved